src/server/utils/get-pr-from-commit.ts

```diff
@@ -72,6 +79,25 @@ export async function getPRDetailsFromCommit(
       continue;
     }
 
+    const reviews = [];
+    let approvedReviewCount = 0;
+    if (prData.reviews && prData.reviews.nodes) {
+      for (const review of prData.reviews.nodes) {
+        if (!review || !review.author || !review.author.login) {
+          continue;
+        }
+
+        if (review.state === 'APPROVED') {
+          approvedReviewCount++;
+        }
+
+        reviews.push({
+          state: review.state,
+          author: review.author.login,
+        });
+      }
+    }
```

Comment:
This doesn't use the same logic that we use on the main dashboard. This could lead to subtle differences. For example, some edge cases which this currently doesn't consider:
 - same reviewer approves multiple times. This would show up as n approved reviews, when really there was only 1.
 - reviewer changes their review - GitHub allows you to approve a change and then request changes.

The main dashboard deals with these cases by a) finding the _current_ review state for a reviewer and b) only looking at one review per reviewer.